pkg/cli/mirror/operator.go

```diff
@@ -222,7 +225,39 @@ func (o *OperatorOptions) renderDCDiff(ctx context.Context, reg *containerdregis
 		}
 	}
 
-	return a.Run(ctx)
+	resultdc, err := a.Run(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	verifyOperatorPkgFound(dic, resultdc)
+
+	return resultdc, nil
+}
+
+// Verify that each of the requested operator packages were found and added to the
+// DeclarativeConfig.
+func verifyOperatorPkgFound(dic action.DiffIncludeConfig, dc *declcfg.DeclarativeConfig) {
+	logrus.Debug("DiffIncludeConfig: ", dic)
+	logrus.Debug("DeclarativeConfig: ", dc)
+
+	for _, pkg := range dic.Packages {
+		logrus.Debug("Checking for package: ", pkg)
+		found := false
+
+		for _, dcpkg := range dc.Packages {
```

Comment:
I would recommend loading the `dc` package names into a map for searching so you don't have to loop through the `dc.Packages` slice for each package in the included config.